ifrs17/Import/Importers.ipynb

```diff
@@ -270,17 +270,17 @@
         {
             "cell_type": "code",
             "source": [
-                "public async Task CommitToDatabase<TData, TPartition> (IWorkspace workspace, Guid partitionId = new Guid(), bool snapshot = true, Expression<Func<TData, bool>> filter = null) ",
+                "public static async Task CommitToIfrsTarget<TData, TPartition> (this IDataSource dataSource, IDataSource targetSource, Guid partitionId = new Guid(), bool snapshot = true, Expression<Func<TData, bool>> filter = null) ",
```

Comment:
use "default"

Reply:
done for the partition. The rest can't be done, see prev reply.